doc/modules/cassandra/pages/operating/audit_logging.adoc

```diff
@@ -150,8 +150,12 @@ auditlogviewer <path1> [<path2>...<pathN>] [options]
     waiting for more records
 `-r,--roll_cycle`::
   How often to roll the log file was rolled. May be;;
-    necessary for Chronicle to correctly parse file names. (MINUTELY,
-    HOURLY, DAILY). Default HOURLY.
+    necessary for Chronicle to correctly parse file names. Some available options are:
+FIVE_MINUTELY, FAST_HOURLY, FAST_DAILY, LargeRollCycles.LARGE_DAILY, LargeRollCycles.XLARGE_DAILY,
+LargeRollCycles.HUGE_DAILY. Deprecated ones still availble but not recommended for new deployments:
+MINUTELY, HOURLY, DAILY
+For more options, refer: net.openhft.chronicle.queue.RollCycles.
+Default is set to FAST_HOURLY
```

Comment:
(same questioning as above)  deserves to also be upstream ?

Reply:
Not yet, check my previous comment

Reply:
Should this also go upstream ?  

I am not asking you, or anyone, to do that.  Just asking if that makes sense.

Reply:
> Should this also go upstream ? 

Now I get it 🤓 
Upstream, Cassandra 5.0 and trunk is still only including 5.23.x

So this change is applicable upstream, but only when we bump from 5.23.x to 5.24.x